MAUI/TreeMap/getting-started.md

```diff
@@ -353,6 +355,8 @@ public class PopulationViewModel
 {% endhighlight %}
 {% endtabs %}
 
+N> For more details on leaf item brush settings, you can refer to [Brush settings](https://help.syncfusion.com/maui/treemap/brush-settings) docummentation.
+
```

Comment:
docummentation - spell error, resolve it all places.

Reply:
Documentation - changed